### ISSUE TITLE
Is this project the implementation of liang-chieh chen's deeplabv3?

### ISSUE THREAD

**Issue by xjock** [Harvey1973/ResNet-deeplabV3#1]
Rethinking Atrous Convolution for Semantic Image Segmentation.
Liang-Chieh Chen George Papandreou Florian Schroff Hartwig Adam

**Comment by Harvey1973** [comment]
Yes It is my attempt for deep lab V2. but it's messy and far from complete

**Comment by xjock** [comment]
Good job!

**Comment by John1231983** [comment]
Any new? Do you achieve the result same paper report?

**Comment by panyuetj** [comment]
How to understand the "block" in the deeplab_v3?   Is it as same as the res_unit or stage in resnet?

**Comment by Harvey1973** [comment]
Sorry I haven't got the time to work on it nowadays , I may pick it up later this year
